node/src/components/storage/lmdb_block_height_store.rs

```diff
@@ -4,10 +4,11 @@ use std::{
     sync::atomic::{AtomicU64, Ordering},
 };
 
+use casper_types::bytesrepr::{self, FromBytes, ToBytes};
```

Comment:
The imports should be grouped as follows, with a blank line between each group:
* std
* third party libs
* CasperLabs libs other than the current crate
* the current crate

Reply:
I blame VSCode for not knowing this convention. I have resolved it as requested.